CIPs/CIP-8/CIP-8.md

```diff
@@ -12,24 +12,24 @@ requires: Doctypes (CIP-5)
 
 ## Simple Summary
 
-This CIP describes the Tile doctype which is a very general document type that has JSON as it's content and is controlled by a [DID](https://w3c.github.io/did-core/).
+This CIP describes the Tile document which is a very general stream type that has JSON as it's content and is controlled by a [DID](https://w3c.github.io/did-core/).
 
 
 ## Abstract
 
-The Tile doctype allows DIDs to create simple JSON documents that use [json-patch](http://jsonpatch.com/) for updating the content of the document. Each Tile is owned by one or multiple DIDs, all of which need to sign the update record. The signed records are encoded using [dag-jose](https://github.com/ceramicnetwork/js-dag-jose) which allows the signatures to be natively encoded directly in [IPLD](ipld.io). Tiles also support adding schemas to the document which means that the json data within the content of the document can be enforced to have a specific format.
+The Tile document allows DIDs to create simple JSON documents that use [json-patch](http://jsonpatch.com/) for updating the content of the document. Each Tile is owned by one or multiple DIDs, all of which need to sign the update record. The signed records are encoded using [dag-jose](https://github.com/ceramicnetwork/js-dag-jose) which allows the signatures to be natively encoded directly in [IPLD](ipld.io). Tiles also support adding schemas to the document which means that the json data within the content of the document can be enforced to have a specific format.
```

Comment:
I wonder if we should remove the language about multiple DIDs for now?  Until we actually support multiple controllers and know what the semantics will be

Reply:
I didn't really look so deeply here. Could definitely do that.